### ISSUE TITLE
Questions about the class LIPParsingEdgeDateset

### ISSUE THREAD

**Issue by zzw1123** [liutinglt/CE2P#19]
Hi, I have noticed the class LIPParsingEdgeDateset in datasets.py is implemented by following method:
```
self.img_ids = [i_id.strip().split() for i_id in open (list_path)] 
if not max_iters==None:
    self.img_ids = self.img_ids * int(np.ceil(float(max_iters) / len(self.img_ids)))
```
max_iters=batch_size*total_iters which means if there are 1000 images in the whole dataset, it's going to be **max_iters** images now, so if I change the batch_size, the total number of images to be trained will be different. Am I right?

**Comment by liutinglt** [comment]
@zzw1123 You are right, we train about 150 epochs to achieve the reported results. Now, please use the updated code.

**Comment by zzw1123** [comment]
@liutinglt Thank you so much! I will update my code.

**Comment by zzw1123** [comment]
@liutinglt I noticed that the size of input image is changed to '384,384', right?

**Comment by liutinglt** [comment]
@zzw1123 The size with '384,384' already can achieve the reported performance. The edge is generated automatically, now you can change it to any size.
